mailpoet/tests/integration/Subscribers/InactiveSubscribersControllerTest.php

```diff
@@ -28,6 +28,7 @@ class InactiveSubscribersControllerTest extends \MailPoetTest {
 
   const INACTIVITY_DAYS_THRESHOLD = 5;
   const PROCESS_BATCH_SIZE = 100;
+  const UNOPENED_EMAILS_THRESHOLD = 1;
```

Comment:
I wonder if a test with `1` is a good test. It seems to be a bit of a "special case". Would it be hard to make the test use the actual `3` (and leave this parameter non-configurable in the method calls)?

Reply:
I will change it, true that 1 might not be the best case. It was the previous number and I didn't want to change all the tests.
It was also a way to test that changing the default also works, in case want to turn it into a setting, but I only need one test with that.